tests/test_survey_schema_translation.py

```diff
@@ -504,3 +504,32 @@ def test_locale_on_load_ulster_scots():
     translation.load("tests/schemas/test_language-eo.po")
 
     assert translation.language == "eo"
+
+
+def test_checkbox_null_label():
+
+    schema_translation = SchemaTranslation()
+    catalog = Catalog(locale=Locale("cy"))
+    catalog.add(
+        "Rugby", "Rygbi",
+    )
+    schema_translation.catalog = catalog
+    schema = SurveySchema(
+        {
+            "question": {
+                "answers": [
+                    {"label": None, "options": [{"label": "Rugby", "value": "Rugby"}]}
```

Comment:
I am slightly confused, why is it None here, it was null in the schema repo?

Reply:
it's because when python parses json files it changes it to `None` as there is no `null` in python, same in runner templated we used `if label == None`.

Reply:
Ugly